### DISCUSSION TITLE
Vague reference to "$combined" in docs

### DISCUSSION THREAD

**Discussion by nathanchere** [starship/starship#5259] [Q&A]
There is a mention in the configuration docs about combined variables, i.e.:

```
When $combined is a shortcut for \[$a$b\], '($combined)' will show nothing only if $a and $b are both None. This works the same as '(\[$a$b\] )'.
```

This is the only reference I can find anywhere to combining variables. Is this an undocumented feature or am I expecting too much? Ideally I'm hoping it's possible to do something like:

```
dev_vars = "[$c$dotnet$golang$java$python$rust]"
```

to clean up the main prompt definitions by being able to have e.g. `([Environmen: $dev_vars])` instead of all the relevant dev environments declared inline.

**Comment by mickimnet** [comment]
I believe this is referring to modules, which have combined variables, like the variable `$all` in the `format` module, or `$all_status` in the `[git_status]` module.
